Apps/APN/ios/Podfile

```diff
@@ -10,8 +10,6 @@ target 'SampleApp' do
   # Flags change depending on the env values.
   flags = get_default_flags()
 
-  pod 'CustomerIO/MessagingPushAPN', '~> 2'
```

Comment:
Notice how I removed this entry from the main app target. Customers no longer need to specify this cocoapod because of the RN autolinking feature that automatically installs all dependencies inside of `customerio-reactnative.podspec` file.